src/commonMethods/commonFunctions.js

```diff
@@ -0,0 +1,353 @@
+import constants from '../commonMethods/constants'
+import moment from 'moment'
+const httpreq = require('http')
+const Client = require('ssh2').Client
+const URL = require('url')
+const http = require('http')
+const os = require('os')
+const path = require('path')
+const httpServer = require('http-server')
+const fs = require('fs')
+const httpclient = require('httpclient')
```

Comment:
Don't mix `import` with `require`
Also I see we have some imports that aren't used (URL, path, httpServer, fs, httpClient) - please review